platform/MCU/STM32F2xx/SPARK_Firmware_Driver/inc/dct.h

```diff
@@ -75,8 +75,8 @@ typedef struct __attribute__((packed)) application_dct {
     uint8_t alt_device_private_key[192];	// alternative device private key
     uint8_t alt_server_public_key[192];
     uint8_t alt_server_address[DCT_SERVER_ADDRESS_SIZE];		// server address info
-
-    uint8_t reserved2[640];
+    uint8_t device_id[12];									// the STM32 device ID
```

Comment:
I added "Device electronic signature" in a comment in the appropriate HAL function since that's the STM name for the unique device ID.